index.html

```diff
@@ -0,0 +1,154 @@
+                        </span>
+                    </div>
+                    <div class="lecture">
+                        <img src="images/program_icon_04.png" alt="">
+                        <p class="sub-heading">Workshop</p>
+                        <span class="text">
+                            try creating your own work using open 
+                            source license rather than just watching at it
+                        </span>
+                    </div>
+                    <div class="lecture">
+                        <img src="images/program_icon_05.png" alt="">
+                        <p class="sub-heading">CC Ignite</p>
+                        <span class="text">
+                            Get opportunities to network us CC affiliate around the 
+                            world and also after summiting 
+                        </span>
+                    </div>
+                </div>
+                <div class="btn-con"><button class="study-btn">join my study</button></div>
+                <li class="program-detail"><a href="#">See the whole program</a></li>
+            </div>
+        </section>
+        <section id="section2">
+            <div class="main-container">
+                <div class="presenters">
+                    <span class="main-program2">Main program</span>
+                    <hr class="hr-line">
+                    <div id="speakers-grid">
+
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="connect-section" id="section3">
+            <div class="main-container">
+                <span class="main-program2">Main program</span>
+                <hr class="hr-line">
+                <div class="connect-img">
+                    <img class="the-img" src="images/mozilla.png" alt="">
+                    <img class="the-img" src="images/googlekorea.png" alt="">
+                    <img class="the-img" src="images/naver.png" alt="">
+                    <img class="the-img" src="images/kakao-2.png" alt="">
+                    <img class="the-img" src="images/airbnb-2.png" alt="">
+                </div>
+            </div>
+        </section>
+        <section class="footer">
+            <div class="main-container">
+                <div class="footer-con">
+                    <div class="my-logo">
+                        <a href="#"><img class="logo" src="images/Ifeanyi_Logo_1B-01-removebg-preview.png" alt=""></a>
+                    </div>
+                    <div>
+                        <p class="down">2022 Creative Commons Korea. Some Rights Reserved.</p>
+                        <p>All content on the website, unless otherwise specified, is available under a Creative Commons Attribution 2.0 Korea License. Please see http://creativecommons.org/licences/by/2.0/kr</p>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Kindly follow the given design by using flexbox to align the content with the logo and the first part of the text  to the left and the text to the right.

Your design
![image](https://user-images.githubusercontent.com/81932478/196735735-666729f3-8952-451e-80b0-e334f9803543.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196735801-e6791300-63e4-41c4-8cb3-0fd27632f272.png)